src/types.ts

```diff
@@ -2,4 +2,5 @@ export interface Quote {
   quote: string;
   author: string;
   likeCount: number;
+  userId?: string;
```

Comment:
Is this userId for the user that created the quote? Better to name it 'createdBy' or 'addedBy'